.github/workflows/tests.yaml

```diff
@@ -20,20 +20,20 @@ jobs:
     name: lint
     runs-on: ubuntu-latest
     steps:
-    - name: Setup Go
-      uses: actions/setup-go@v2
-      with:
-        go-version: '1.16'
-    - name: Install goimports
-      run: go get golang.org/x/tools/cmd/goimports
-    - name: Checkout code
-      uses: actions/checkout@v2
-    - run: goimports -w .
-    - run: go mod tidy
-    - name: Verify no changes from goimports and go mod tidy. If you're reading this and the check has failed, run `goimports -w . && go mod tidy`.
-      run: git diff --exit-code 
-    - name: check for mismatch
-      shell: bash
-      run: |
-        VERSION=$(cat version.txt)
-        grep -w \"$VERSION\" cmd/cloud_sql_proxy/cloud_sql_proxy.go || exit 1
+      - name: Setup Go
```

Comment:
nit: whitespace

Reply:
Which whitespace? The indent is how the yaml formatter seems to think it should be

Reply:
Which whitespace? The indent is how the yaml formatter seems to think it should be

Reply:
OK. I see the VS Code formatter does this.